AEPIdentity/Sources/IdentityState.swift

```diff
@@ -185,7 +185,7 @@ class IdentityState {
 
         // check config
         if !canSyncForCurrentConfiguration(config: currentEventValidConfig) {
-            // TOOD: Add log
+            // TODO: Add log
```

Comment:
Can we just add the log?

Reply:
haha, i just saw one `TOOD` and did a find/replace.  i'll add the log and get rid of the `TODO:`